apps/climatemappedafrica/src/lib/data/blockify/explore-page.js

```diff
@@ -5,10 +5,10 @@ import {
 
 async function explorePage({ block: { slugs }, hurumap }) {
   const {
-    initialLocation,
+    rootGeography,
     page: { value },
   } = hurumap;
-  const { name } = initialLocation;
+  const { code: name, rootGeographyHasData: pinRootGeography } = rootGeography;
```

Comment:
I would have kept `rootGeography` and `rootGeographyHasData` names instead of `initialLocation` and `pinRootGeography` or `pinInitialLocation`.